buffling/signal_to_noise_ratio.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2020 Google LLC
```

Comment:
Please add a unit testing file for this module.

Reply:
added